engine/coderunner/v2/runner.go

```diff
@@ -33,6 +33,15 @@ func (cr *CodeRunner) Run(props *RunnerProps) (*RunnerOutput, error) {
 		Cputime:   runtime.DefaultCputime,
 	}
 
+	// Language-specific modifications
+	// Rust has large binaries, even for simple applications
+	//
+	// ... is there a better way to do this without switching on names?
+	switch language.Name {
+	case "rust":
+		compileCommands.Fsize = 1 << 25 // 32 mB
+	}
```

Comment:
One small note is that rust executables seem to be pretty large; without some more extensive configurations, a simple "hello world" already came out to be 11 mB. I added a stipulation in the engine config that checks if the language is set to rust, and if so, it increases the max file size. 

This approach works, but it seems a bit fragile

Reply:
That's true and a good callout. I saw some similar behavior with Go binaries if you try including too many libraries.